credentials/common.md

```diff
@@ -112,6 +112,12 @@ defines value type
 ```
  "serialization": Index
 ```
+# secret
+
+A secret number 
```

Comment:
add the purpose of secret number. what is it for?

Reply:
Done, thanks!